### PR TITLE
Dashboard view

### LINKED ISSUE #39: Dashboard view
### Description:
As any user, I want to be able to see any news and/or advertisements so that I can learn more about different offers.

### Assumptions:
* Main window is initialized

### Estimate:
* 2

### Tasks:
* Create a new view
* Add content to the view

### Tests:
* Check if all the content is in the view

### Definition of Done:
The user is able to see content on the dashboard.

### PR DESCRIPTION
**Pull request by ChrisRzech**
Closes #39 

### Key features
* Dashboard view usable in `MainWindow`
* Splash message system that randomizes whenever the view resets
* Image slideshow that changes image every few seconds (currently 2 seconds)

### Future Improvements
* Definitely more content, looks empty

### Notes
Dashboard uses the CSV file parser to parse the splash file (which is a `.csv` for this very reason)

### CODE REVIEW

**Review by Roger-Chavez — CHANGES_REQUESTED** [review]
Please change splash messages to be something baseball related. For example baseball trivia.

File: src/widgets/imageslideshow.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * itself. If @a pause has been called, this function
+ * does nothing.
+ */
+void ImageSlideshow::loop()
+{
+    if(!m_play)
+        return;
+
+    setPixmap(m_images[m_currentIndex]);
+
```
Comment (WakkaTrout) [inline src/widgets/imageslideshow.cpp]: Just squish these 3 lines and you good

**Review by WakkaTrout — CHANGES_REQUESTED** [review]

**Review by yosmo78 — APPROVED** [review]
looks good, works goods, even smells good

**Review by WakkaTrout — APPROVED** [review]
I can't find anything bad and it runs fine on my computer, so approved

**Review by Roger-Chavez — APPROVED** [review]

**Pull request merged**